content/en/includes/agent/agent-compat-matrix.md

```diff
@@ -1,8 +1,8 @@
 > The Armory Agent is in early access. For more information about using this feature, [contact us](https://www.armory.io/contact-us/).
 
-The Armory Agent is compatible with Armory Enterprise and open source Spinnaker. It consists of a lightweight service that you deploy on Kubernetes and a plugin that you install into Spinnaker. Your Clouddriver service must use a MySQL-compatible database. See the {{< linkWithTitle "clouddriver-sql-configure.md" >}} guide for instructions.
+The Armory Agent is compatible with Armory Enterprise and open source Spinnaker. It consists of a lightweight Agent service that you deploy on Kubernetes and a plugin that you install into Clouddriver. Your Clouddriver service must use a MySQL-compatible database. See the {{< linkWithTitle "clouddriver-sql-configure.md" >}} guide for instructions.
```

Comment:
I'd expect a sentence like this to be near the start of the main Agent page 

```
It consists of a lightweight Agent service that you deploy on Kubernetes and a plugin that you install into Clouddriver.

Reply:
Agree. Moved out of the compat include and onto the index page.